frontend/public/components/sidebars/explore-type-sidebar.tsx

```diff
@@ -154,10 +161,10 @@ export const ExploreType: React.FC<ExploreTypeProps> = (props) => {
                   >
                     <CamelCaseWrap value={name} />
                     &nbsp;
-                    <small>
+                    <Content component={ContentVariants.small}>
```

Comment:
Changed to `<Content>` since this is inside an existing `<Content>` on line 132.